src/components/LocationToCompare.js

```diff
@@ -36,14 +38,16 @@ const disclaimerText = {
   marginTop: 10,
 };
 
-const LocationCompare = () => {
+const LocationCompare = ({showButton}) => {
   const comparisonFeatures = useSelector(getComparisonFeatures);
+  const history = useHistory();
+
   return (
     <div style={root}>
     <div style={listRoot}>
       {comparisonFeatures.map((feature) => {
         return (
-          <div style={featureRoot} key={feature.primary}>
+          <div style={featureRoot} key={feature.properties.SA2_MAIN16}>
             <div>{feature.primary || feature.properties["SA2_NAME16"]}</div>
```

Comment:
This can be simplified to `{feature.properties["SA2_NAME16"]}`. The enclosing div and check for feature.primary are unnecessary.